lib/index.js

```diff
@@ -10368,6 +10368,15 @@ async function run() {
     await execShellCommand(`echo 'set +e' >/tmp/tmate.bashrc`);
     const setDefaultCommand = `set-option -g default-command "bash --rcfile /tmp/tmate.bashrc" \\;`;
 
+    const tmateConfPath = external_path_default().join(external_os_default().homedir(), ".tmate.conf")
```

Comment:
Why not use `/tmp/tmate.conf`?

Reply:
`/tmp` directory hasn't not a not UNIX-like OS (e.g. Windows). So I used `${HOME}` dir.
But I found that this action uses MSYS2 and MSYS2 can uses `/tmp`. 

I will change to use `/tmp/tmate.conf`.

Reply:
> Also, since this is relevant only when starting the session, we could potentially pass the custom settings via the command-line, yes?

I see. I will change to add `newSessionExtra`.